src/devTools/editor/tabs/editTab/editorNodeLayout/EditorNodeLayout.tsx

```diff
@@ -109,37 +90,64 @@ const EditorNodeLayout: React.FC<{
             };
           }
 
+          const showAddBlock = index < finalIndex || showAppend;
+          const showDuplicate = showAddBlock && index > 0;
+
           return (
-            <React.Fragment key={index}>
-              {nodeId !== FOUNDATION_NODE_ID && (
-                <BlockModal
-                  bricks={relevantBlocksToAdd}
-                  renderButton={(onClick) => renderActions(onClick)}
-                  recommendations={recommendations}
-                  selectCaption={addBrickCaption}
-                  onSelect={(block) => {
-                    addBlock(block, nodeId);
-                  }}
-                />
-              )}
+            <React.Fragment key={nodeId}>
               <EditorNode
                 active={nodeId === activeNodeId}
                 canMoveAnything={canMoveAnything}
                 {...nodeProps}
               />
+              <div
+                className={cx(styles.actions, {
+                  [styles.finalActions]: index === finalIndex,
+                })}
+              >
+                {showAddBlock && (
+                  <BlockModal
+                    bricks={relevantBlocksToAdd}
+                    renderButton={(onClick) => (
+                      <TooltipIconButton
+                        name={`add-node-${index}`}
+                        icon={faPlusCircle}
+                        onClick={onClick}
+                        tooltipText="Add a brick"
+                      />
+                    )}
+                    recommendations={recommendations}
+                    selectCaption={addBrickCaption}
+                    onSelect={(block) => {
+                      addBlock(block, index);
+                    }}
+                  />
+                )}
+                {showDuplicate && (
+                  <TooltipIconButton
+                    name={`duplicate-node-${index}`}
+                    icon={faRecycle}
+                    onClick={() => {
+                      duplicateBlock(nodeId);
+                    }}
+                    tooltipText={`Duplicate block "${title}"`}
```

Comment:
s/block/brick/ everywhere?

```suggestion
                    tooltipText={`Duplicate brick "${title}"`}
```